source_pubsub/source.go

```diff
@@ -0,0 +1,96 @@
+package source
```

Comment:
FYI: there's a lint check for a header here. I assume it's due to missing copyright